docs/lp_token_bridging.md

```diff
@@ -8,7 +8,8 @@ in order to participate in bonding.
+| Solana      | [Orca](https://www.orca.so/liquidity?address=5dMKUYJDsjZkAD3wiV3ViQkuq9pSmWQ5eAzcQLtDnUT3)                                 | WSOL-OLAS   | [CeZ77ti3nPAmcgRkBkUC1JcoAhR8jRti2DHaCcuyUnzR](https://solscan.io/token/CeZ77ti3nPAmcgRkBkUC1JcoAhR8jRti2DHaCcuyUnzR)                                                         | [0x3685B8cC36B8df09ED9E81C1690100306bF23E04](https://etherscan.io/address/0x3685b8cc36b8df09ed9e81c1690100306bf23e04)                                                                                                                | [Wormhole Portal](https://portalbridge.com/advanced-tools/#/transfer)  |
 | Arbitrum    | [Balancer](https://app.balancer.fi/#/arbitrum/pool/0xaf8912a3c4f55a8584b67df30ee0ddf0e60e01f80002000000000000000004fc)     | OLAS-WETH   | [0xAF8912a3C4f55a8584B67DF30ee0dDf0e60e01f8](https://arbiscan.io/token/0xaf8912a3c4f55a8584b67df30ee0ddf0e60e01f8)                | [0x36B203Cb3086269f005a4b987772452243c0767f](https://etherscan.io/token/0x36B203Cb3086269f005a4b987772452243c0767f) | [Wormhole Portal](https://portalbridge.com/advanced-tools/#/transfer)  |
 | Optimism    | [Balancer](https://app.balancer.fi/#/optimism/pool/0x5bb3e58887264b667f915130fd04bbb56116c27800020000000000000000012a)     | WETH-OLAS   | [0x5bb3e58887264b667f915130fd04bbb56116c278](https://optimistic.etherscan.io//address/0x5bb3e58887264b667f915130fd04bbb56116c278) | [0x2FD007a534eB7527b535a1DF35aba6bD2a8b660F](https://etherscan.io/token/0x2FD007a534eB7527b535a1DF35aba6bD2a8b660F) | [Wormhole Portal](https://portalbridge.com/advanced-tools/#/transfer)  |
-| Base        | [Balancer](https://app.balancer.fi/#/base/pool/0x5332584890d6e415a6dc910254d6430b8aab7e69000200000000000000000103)     | OLAS-USDC   | [0x5332584890d6e415a6dc910254d6430b8aab7e69](https://basescan.org/address/0x5332584890d6e415a6dc910254d6430b8aab7e69)             | [0x9946d6FD1210D85EC613Ca956F142D911C97a074](https://etherscan.io/token/0x9946d6FD1210D85EC613Ca956F142D911C97a074) | [Wormhole Portal](https://portalbridge.com/advanced-tools/#/transfer)  |
\ No newline at end of file
+| Base        | [Balancer](https://app.balancer.fi/#/base/pool/0x5332584890d6e415a6dc910254d6430b8aab7e69000200000000000000000103)         | OLAS-USDC   | [0x5332584890d6e415a6dc910254d6430b8aab7e69](https://basescan.org/address/0x5332584890d6e415a6dc910254d6430b8aab7e69)             | [0x9946d6FD1210D85EC613Ca956F142D911C97a074](https://etherscan.io/token/0x9946d6FD1210D85EC613Ca956F142D911C97a074) | [Wormhole Portal](https://portalbridge.com/advanced-tools/#/transfer)  |
+| Celo        | [Ubeswap](https://app.ubeswap.org/#/swap)          | CELO-OLAS       | [0x2976Fa805141b467BCBc6334a69AffF4D914d96A](https://celoscan.io/address/0x2976Fa805141b467BCBc6334a69AffF4D914d96A)             | [0xC085F31E4ca659fF8A17042dDB26f1dcA2fBdAB4](https://etherscan.io/token/0xc085f31e4ca659ff8a17042ddb26f1dca2fbdab4) | [Wormhole Portal](https://portalbridge.com/advanced-tools/#/transfer)  |
```

Comment:
Celo info added

Reply:
can the swap url not point directly to our pair? see how base points to the pool

Reply:
The Base is still pointing to the pool UI. I've spent a lot of time trying to do it at least how the Uniswap is done, but it doesn't work. If it's fine to have just a pair address, we can do this.

Reply:
https://info-v2.ubeswap.org/pair/0x2976Fa805141b467BCBc6334a69AffF4D914d96A maybe?